api/proto/config/vm.proto

```diff
@@ -26,11 +26,21 @@ message VmConfig {
   string ramdisk = 2;
   uint32 memory = 3;
   uint32 maxmem = 4;
+  // The amount of the VCPUs in the VM.
   uint32 vcpus = 5;
+  // Currently is not handled by EVE.
```

Comment:
Seems, you are right, we do not propagate the field from config in parseAppInstanceConfig function. It is quite strange, should we modify it in https://github.com/lf-edge/eve/pull/2937 while it is open?

Reply:
Ok, will do.